src/assembler.cpp

```diff
@@ -1242,6 +1242,9 @@ int main(int argc, char* argv[])
                         constant.b = (uint32_t)values[0] != 0;
                         break;
                     }
+
+                    default:
```

Comment:
I will need much more time and energy to fully understand this big function if I want to be sure that can or cannot happen. For the time being I just made what happens in the default case explicit.
It doesn't changes the behavior of this switch.